vmdb/app/models/metric/capture.rb

```diff
@@ -4,7 +4,9 @@ module Metric::Capture
   REALTIME_PRIORITY = HOURLY_PRIORITY = DAILY_PRIORITY = MiqQueue::NORMAL_PRIORITY
   HISTORICAL_PRIORITY = MiqQueue::LOW_PRIORITY
 
-  CAPTURE_COLS = Metric.columns_hash.collect { |c, h| c.to_sym if h.type == :float && c[0, 7] != "derived" }.compact
+  def self.capture_cols
+    Metric.columns_hash.collect { |c, h| c.to_sym if h.type == :float && c[0, 7] != "derived" }.compact
+  end
```

Comment:
@matthewd Just noticed this, but we should probably should be memoizing this, otherwise it gets recalculated in the loops every time.